pyproject.toml

```diff
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Contributors to the OpenEXR Project.
+
+[build-system]
+requires = ["scikit-build-core==0.8.0"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "OpenEXR"
+dynamic = ["version"]
+
+description="Python bindings for the OpenEXR image file format"
+readme = "src/wrappers/python/README.md"
+authors = [
+  { name="Contributors to the OpenEXR project", email="info@openexr.com" },
+]
+requires-python = ">=3.7"
+
+[project.urls]
+"Homepage" = "https://openexr.com"
+"Source" = "https://github.com/AcademySoftwareFoundation/OpenEXR"
+"Bug Tracker" = "https://github.com/AcademySoftwareFoundation/OpenEXR/issues"
+
+[project.optional-dependencies]
+test = ["pytest"]
+
+[tool.scikit-build]
+wheel.expand-macos-universal-tags = true
+wheel.packages = ["src/wrappers/python"]
```

Comment:
```suggestion
```
This tries to package a package called `python`, but in this case, all the files (include `Imath.py` are installed from the CMakeLists.txt file. `wheel.packages` is only needed when you want to let scikit-build-core install the python files.

In this case, we can just remove it.